### PR TITLE
#12 issue solved - Changes done in the UI of  home page better user Interaction.

### PR DESCRIPTION
**Pull request by SoumyaU25**
## 🔨 Changes Made

- 
- 
- 

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x] I have followed all the guidlines mentioned in CONTRIBUTING.md.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] I have tested it locally and it works fine.
- [x] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [ ] Bug fix (non-breaking change which fixes an issue) 🐛
- [ ] New feature (non-breaking change which adds functionality) ✨
- [x] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

- Fixes #

## 📸 Screenshots (if applicable)

r1
r3
r4

---

*Thank you for contributing to Research Nexus! We look forward to reviewing your PR and getting it merged!*

### CODE REVIEW

**Review by Harshdev098 — CHANGES_REQUESTED** [review]
Is something wrong with this,
image

Please increase the font-size of the navbar lists and the minimize the padding and side margin

**Comment by SoumyaU25** [comment]
Okay @Harshdev098 I'll do that

**Comment by SoumyaU25** [comment]
Hello @Harshdev098 I have done the changes . Please Check it out.

**Comment by Harshdev098** [comment]
Are the background image that you have set in the screenshot is not working 

image

**Comment by SoumyaU25** [comment]
@Harshdev098 actually it's working fine at my end, I have set the right path for it.
r5

File: public/css/main_page_style.css [hunk 1]
```diff
@@ -1,252 +1,195 @@
...
-
-/* header styling  */
+    font-family: sans-serif;
+  }
+  
+  /* header styling */
+  .banner {
+    width: 100%;
+    height: 100vh;
+    background-image: linear-gradient(rgba(8, 23, 82, 0.75), rgba(30, 2, 30, 0.75)), url('public/images/calm.jpg');
```
Comment (Harshdev098) [inline public/css/main_page_style.css:12]: edit the url with this `../images/calm.jpg`

File: public/main_page.html [hunk 2]
```diff
@@ -63,7 +64,51 @@ <h2 style="padding: 10px;font-size: 44px;color: rgb(202, 60, 60);">Explore Resea
...
         </section>
-    </main>
+    </main> -->
+
+    <div class="banner">
+       
+        <div class="nav">
+          <img class="logo" src="images/logo.png" alt="">
+          <form class="search-bar">
+            <input type="text" placeholder="Search...">
```
Comment (Harshdev098) [inline public/main_page.html:74]: Increase the input size of search box something like 20vw

File: public/main_page.html [hunk 3]
```diff
@@ -63,7 +64,51 @@ <h2 style="padding: 10px;font-size: 44px;color: rgb(202, 60, 60);">Explore Resea
...
+            <button type="submit">Search</button>
+          </form>
+          <ul>
+             <li><a href="">Home</a></li>
+             <li><a href="">About</a></li>
+             <li><a href="">Add Topic</a></li>
+             <li onmouseover="show()" onmouseout="hide()"><a href=""
+                 title="View Notifications"><i id="firsticon" class="fa-sharp fa-regular fa-bell"
```
Comment (Harshdev098) [inline public/main_page.html:82]: The notification btn is not working as expected onmouseover and onmouseout properties

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

**Comment by SoumyaU25** [comment]
@Harshdev098 I have done all the changes as you said
changed the path for background image [x]
the input size of search box is increased about 22vw. [x]
The notification btn is working fine now. [x]
r6

**Comment by Harshdev098** [comment]
@SoumyaU25 Delete the .vscode folder

**Comment by SoumyaU25** [comment]
Okay @Harshdev098

**Comment by SoumyaU25** [comment]
Done @Harshdev098

**Pull request merged**